mobile/src/libs/components/toast/toast.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Toast, { ErrorToast } from 'react-native-toast-message';
+
+import { AppColor } from '#libs/enums/enums';
+
+const toastConfig = {
+  error: (properties: object): React.ReactNode => (
+    <ErrorToast
+      {...properties}
+      style={{ borderColor: AppColor.RED_100 }}
+      text1Style={{
+        fontSize: 15,
+      }}
+      text2Style={{
+        color: AppColor.GRAY_400,
```

Comment:
you can create file styles.ts near your toast.tsx component and write all styles there